packages/expo-file-system/ios/EXFileSystem/EXFileSystem.m

```diff
@@ -1012,6 +1012,15 @@ - (BOOL)checkIfFileDirExists:(NSString *)path
   return [self _checkIfFileExists:dir];
 }
 
+- (NSURL *)percentEncodeURIString:(NSString *)uri
+{
```

Comment:
This method needs documentation, including a reference to the RFC. "Percent-encoding" a string could mean many different things. Our use case is non-obvious.

Additionally, iOS methods that are mostly for producing a value should be named using noun phrases like `URLFromPercentEncodedURIString:`

Reply:
That makes sense, I added doc comments, `nullable` and `nonnull` annotations and renamed the function to `percentEncodedURLFromURIString`